### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by alinebiath**
Improvements to replace variables names, remove unnecessary comments and reorganize code.

### CODE REVIEW

File: resource/keywords_bdd.robot [hunk 1]
```diff
@@ -77,23 +77,15 @@ I proceed to checkout
...
     selenium.Element Should Contain                                                  css:td.cart_description > p:nth-child(1) > a:nth-child(1)         Faded Short Sleeve T-shirts
-    #obter a posição horizontal do botão "Proceed to checkout"
-    ${total_position_y}                                                              selenium.Get Horizontal Position                                  xpath=//*[@id="center_column"]/p[2]/a[1]
-    # subtrai 550 da posição obtida
-    ${total_position_y}                                                              Evaluate                                                          ${total_position_y} - 550
-    #executa o scroll de 0 até a posição - 550
-    selenium.Execute Javascript                                                      window.scrollTo(0,${total_position_y})
+    Set Test Variable                                                                ${locator}                                                        xpath=//*[@id="center_column"]/p[2]/a[1]
+    Scroll to specifc Position                                                       ${locator}
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: @alinebiath I suggest undo this change and add a comment explaining the motivation for that workaround.

File: resource/keywords_bdd.robot [hunk 2]
```diff
@@ -77,23 +77,15 @@ I proceed to checkout
...
 I access the shopping cart
     selenium.Click Element                                                           xpath=/html/body/div/div[1]/header/div[3]/div/div/div[3]/div/a
     selenium.Wait Until Page Contains Element                                        xpath=//*[@id="center_column"]                                    timeout=8s
-    #obter a posição horizontal do botão "Proceed to checkout"
-    ${total_position_y}                                                              selenium.Get Horizontal Position                                  xpath=//*[@id="center_column"]/p[2]/a[1]
-    # subtrai 550 da posição obtida
-    ${total_position_y}                                                              Evaluate                                                          ${total_position_y} - 550
-    #executa o scroll de 0 até a posição - 550
-    selenium.Execute Javascript                                                      window.scrollTo(0,${total_position_y})
+    Set Test Variable                                                                ${locator}                                                        xpath=//*[@id="center_column"]/p[2]/a[1]
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: @alinebiath, What library alias this keyword belongs to?

File: resource/keywords_bdd.robot [hunk 3]
```diff
@@ -1,9 +1,9 @@
 *** Settings ***
-# para testar um aplicação web
+# para testar um aplicação web.
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot:2]: @alinebiath I suggest add a tag/table Documentation and put all url address for each library. What do you think?

File: resource/keywords_bdd.robot [hunk 4]
```diff
@@ -90,18 +79,22 @@ I proceed to checkout
...
 
-
 # Scenario 05: Remove products
-
 I access the shopping cart
     selenium.Click Element                                                           xpath=/html/body/div/div[1]/header/div[3]/div/div/div[3]/div/a
     selenium.Wait Until Page Contains Element                                        xpath=//*[@id="center_column"]                                    timeout=8s
+    #obter a posição horizontal do botão "Proceed to checkout"
     ${total_position_y}                                                              selenium.Get Horizontal Position                                  xpath=//*[@id="center_column"]/p[2]/a[1]
+    # subtrai 550 da posição obtida
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: This 550 represents an offset? It's a workaround/trick for some problem?

File: resource/keywords_bdd.robot [hunk 5]
```diff
@@ -65,10 +65,8 @@ I add to shopping cart the listed product
     selenium.Page Should Contain Element                                             xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]
     selenium.Mouse Over                                                              xpath=//*[@id="center_column"]/ul/li/div
     selenium.Wait Until Element is Visible                                           xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]       timeout=8s
-    #obter a posição horizontal do botão "Add to cart"
-    ${btn_position_y}                                                                selenium.Get Horizontal Position                                  xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]
-    #executar comando de scroll de 0 até a posíção do botão "Add to cart"
-    selenium.Execute Javascript                                                      window.scrollTo(0,${btn_position_y})
+    Set Test Variable                                                                ${locator}                                                        xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: What is the alias for that keywork?

File: resource/keywords_bdd.robot [hunk 6]
```diff
@@ -14,132 +14,134 @@ ${BROWSER}    firefox
...
-
-###Scenario 03: List products
 I select the "WOMEN" category in the main menu categories
-    selenium.Mouse Over                                                              css:#block_top_menu > ul > li:nth-child(1) > a
+    selenium.Mouse Over                                                              css=#block_top_menu > ul > li:nth-child(1) > a
 
 I select the sub-category "${product}"
-    selenium.Wait Until Element is Visible                                           xpath://*[@id="block_top_menu"]/ul/li[1]/ul                       timeout=8s
-    selenium.Click Element                                                           link:${product}
+    selenium.Wait Until Element is Visible                                           xpath=//*[@id="block_top_menu"]/ul/li[1]/ul                             timeout=12s
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: What do think about configure a new default timeout to entire test case?

File: resource/keywords_bdd.robot [hunk 7]
```diff
@@ -14,132 +14,134 @@ ${BROWSER}    firefox
...
-    selenium.Wait Until Page Contains Element                                        css:#center_column > h1
-    selenium.Element Should Contain                                                  css:#center_column > ul > li > div > div.right-block > h5 > a     Faded Short Sleeve T-shirts
     selenium.Mouse Over                                                              xpath=//*[@id="center_column"]/ul/li/div
-    selenium.Wait Until Element is Visible                                           xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]       timeout=8s
-    Set Test Variable                                                                ${locator}                                                        xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]
+    selenium.Wait Until Element is Visible                                           xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]             timeout=8s
+    Set Test Variable                                                                ${locator}                                                              xpath=//*[@id="center_column"]/ul/li/div/div[2]/div[2]/a[1]
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: What is the alias for that keyword?

File: resource/keywords_bdd.robot [hunk 8]
```diff
@@ -14,132 +14,134 @@ ${BROWSER}    firefox
...
-    Scroll to specifc Position                                                       ${locator}
-
-# Scenario 05: Remove products
 I access the shopping cart
     selenium.Click Element                                                           xpath=/html/body/div/div[1]/header/div[3]/div/div/div[3]/div/a
-    selenium.Wait Until Page Contains Element                                        xpath=//*[@id="center_column"]                                    timeout=8s
-    Set Test Variable                                                                ${locator}                                                        xpath=//*[@id="center_column"]/p[2]/a[1]
+    selenium.Wait Until Page Contains Element                                        xpath=//*[@id="center_column"]                                          timeout=8s
+    Set Test Variable                                                                ${locator}                                                              xpath=//*[@id="center_column"]/p[2]/a[1]
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: Alias.

File: resource/keywords_bdd.robot [hunk 9]
```diff
@@ -14,132 +14,134 @@ ${BROWSER}    firefox
...
 
 I fill the required fields
     selenium.Click Element                                                           id=id_gender1
     ${first_name}=                                                                   faker.First Name
-    selenium.Input Text                                                              id=customer_firstname                                             ${first_name}
+    selenium.Input Text                                                              id=customer_firstname                                                   ${first_name}
     ${last_name}=                                                                    faker.Last Name
-    selenium.Input Text                                                              id=customer_lastname                                              ${last_name}
+    selenium.Input Text                                                              id=customer_lastname                                                    ${last_name}
     ${passwd}=                                                                       faker.Password
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: Would be password instead of passwd?

File: resource/keywords_bdd.robot [hunk 10]
```diff
@@ -14,132 +14,134 @@ ${BROWSER}    firefox
...
-    selenium.Input Password                                                          id=passwd                                                         ${passwd}
+    selenium.Input Password                                                          id=passwd                                                               ${passwd}
     ${birth_day}=                                                                    faker.Day Of Month
-    ${birth_day_ns}=                                                                 string.Strip String                                               ${birth_day}                                                   mode=left           characters=0
-    selenium.Select From List By Value                                               id=days                                                           ${birth_day_ns}
+    ${birth_day_ns}=                                                                 string.Strip String                                                     ${birth_day}                                                   mode=left           characters=0
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: Please, improve that variable name.

File: resource/keywords_bdd.robot [hunk 11]
```diff
@@ -14,132 +14,134 @@ ${BROWSER}    firefox
...
+    selenium.Select From List By Value                                               id=days                                                                 ${birth_day_ns}
     ${birth_month}=                                                                  faker.Month
-    ${birth_month_ns}=                                                               string.Strip String                                               ${birth_month}                                                 mode=left           characters=0
-    selenium.Select From List By Value                                               id=months                                                         ${birth_month_ns}
+    ${birth_month_ns}=                                                               string.Strip String                                                     ${birth_month}                                                 mode=left           characters=0
```
Comment (jaelsonwagner) [inline resource/keywords_bdd.robot]: Please, improve that variable name.

File: resource/keywords_bdd.robot (hunk 1 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.

File: resource/keywords_bdd.robot (hunk 2 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Instated Builtin library.

File: resource/keywords_bdd.robot (hunk 5 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.

File: resource/keywords_bdd.robot (hunk 6 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.

File: resource/keywords_bdd.robot (hunk 7 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.

File: resource/keywords_bdd.robot (hunk 8 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.

File: resource/keywords_bdd.robot (hunk 9 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.

File: resource/keywords_bdd.robot (hunk 10 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.

File: resource/keywords_bdd.robot (hunk 11 above)
Comment (alinebiath) [reply resource/keywords_bdd.robot]: Done.
